app/models/access.rb

```diff
@@ -0,0 +1,38 @@
+class Access < ApplicationRecord
+  ALLOWED_RESOURCE_TYPES = %w[Organization FacilityGroup Facility].freeze
+
+  belongs_to :user, required: true
+  belongs_to :resource, polymorphic: true, optional: true
```

Comment:
With this model, if I need to be an admin for Bathinda and Mansa, would my `User` record have two `Access` records? Both with `role: "admin"`, and one with each FG as the `resource`?